feature-libs/user/profile/components/update-password/update-password-component.service.ts

```diff
@@ -69,10 +92,17 @@ export class UpdatePasswordComponentService {
     const oldPassword = this.form.get('oldPassword')?.value;
     const newPassword = this.form.get('newPassword')?.value;
 
-    this.userPasswordService.update(oldPassword, newPassword).subscribe({
-      next: () => this.onSuccess(),
-      error: (error: Error) => this.onError(error),
-    });
+    if (this.featureConfigService?.isLevel('6.3')) {
```

Comment:
Why don't we have this feature flag in the `onError` method rather than making a new `onNewError` method?

Reply:
@Zeyber ,Since it is api change, we can not use feature flag on api level.